csfieldguide/static/interactives/scene-editor/js/scene-editor.js

```diff
@@ -902,3 +920,36 @@ function createTinyaxisMesh() {
   var object = new THREE.Mesh( geometry, material );
   return object;
 }
+
+
+/** Called on a paste event. Handles populating inputs copied from matrix simplifier 
+ *  or simply validates pasted input if it appears it hasn't been copied from
+ *  matrix simplifier.
+ */
+function populateInputsFromPaste(inputDiv) {
```

Comment:
We need to let the user know they can paste that MS code into any one of the boxes. Consider adding some text-muted instruction

Reply:
This isn't true when we allow them to copy only the matrix or only the vector part.
I added some text to try clarify but happy to hear suggestions.

Also I should probably add some sort of instruction on MS about how to only copy the matrix part or only the vector part, but unsure how to phrase it nicely right now.

Reply:
My thinking is just explain that the user can paste it all (i.e. the copy to clipboard 'all') into any box.
The people who want to will try different things themselves; I'm saying it's good to account for other likely inputs but that doesn't mean we have to 'officially' support them